frontend/src/components/CanvasPanel.vue

```diff
@@ -23,15 +26,24 @@ export default {
     BrushSelector
   },
 
-  mixins: [BrushStyleFactory],
+  mounted: function() {
+    EventBus.$on("brushStroke", this.onBrushStroke);
+  },
 
   computed: {
     defaultBrushStyle() {
-      return this.createBrushStyle(this.sizes[0], this.colors[0]);
+      return createBrushStyle(this.sizes[0], this.colors[0]);
+    }
+  },
+
+  methods: {
+    onBrushStroke: function(brushStroke) {
+      let message = createBrushStrokeMessage(brushStroke);
+      ArcadeWebSocket.send(message);
```

Comment:
I put the code to send the message in this class because I'm treating it as the "entry/exit point" for canvas/brush communication with the rest of the code. Let me know what you think about this.

Reply:
I think that's a good idea for now. I like how it's tied in with the Events so I think it should be good